roles/ceph-common/files/cephstable.asc

```diff
@@ -0,0 +1,31 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
This should be pulled from the repo itself.  This is prone to change and once it does, this will immediately break.